muxpi/nuc/ethup.sh

```diff
@@ -0,0 +1,18 @@
+ #!/bin/bash
+ DUT_IP="$1"
+ n=1 
+ max=5
+ delay=50
+ while true; do
+    ping -c 1 -W 1 $DUT_IP && break || {
```

Comment:
may be use of parentheses like below would be less confusing, if I understand the logic correctly.

(ping -c 1 -W 1 $DUT_IP && break) || {......}

Reply:
You might already have discussed this but just in case: In bash use of parentheses will the command in a sub shell. For instance `(cd /tmp)` will not change the working directory after the command has finished.

Not that there is anything wrong what you are doing currently. Just want you to be aware of possible weirdness :)

Reply:
Thanks, now I know)